include/fixed_containers/fixed_set_adapter.hpp

```diff
@@ -254,10 +253,10 @@ class FixedSetAdapter
         TableIndex idx = table().opaque_index_of(key);
         if (!table().exists(idx))
         {
-            return false;
+            return 0;
         }
         table().erase(idx);
-        return true;
+        return 1;
```

Comment:
should this function return bool instead?

Reply:
The reason for returning an integer is that multiples can have numbers higher than 1:
https://en.cppreference.com/w/cpp/container/multimap/erase